test/NuGet.Clients.Tests/NuGet.VisualStudio.Implementation.Test/SolutionExplorer/Models/AssetsFileDependenciesSnapshotTests.cs

```diff
@@ -0,0 +1,61 @@
+using System.Linq;
+using NuGet.ProjectModel;
+using NuGet.VisualStudio.SolutionExplorer.Models;
+using Xunit;
+
+namespace NuGet.VisualStudio.Implementation.Test.SolutionExplorer.Models
+{
+    public class AssetsFileDependenciesSnapshotTests
+    {
+        [Fact]
+        public void ParseLibraries_IgnoreCaseInDependenciesTree_Succeeds()
```

Comment:
In general, I like having at least one negative test case whenever possible.

Reply:
I will merge this PR into `dev` now and submit a follow up by adding a negative test case.

Reply:
NuGet/Home#9971